ansible/group_vars/nimbus.prater.yml

```diff
@@ -7,6 +7,7 @@ beacon_node_repo_branch: >-
        ((node.branch == "libp2p") | ternary("nim-libp2p-auto-bump-unstable", node.branch))
      ) }}
 beacon_node_nim_commit: '{{ node.get("nim_commit", "") }}'
+beacon_node_build_nim_flags: '-d:noSignalHandler {{ node.get("nim_flags", "") }}'
```

Comment:
I the `-d:testnet_servers_image` no longer necessary?

Reply:
We do add it by default to builds:
https://github.com/status-im/infra-role-beacon-node-linux/blob/323a957d14891447843ba76fee06aae9d120fa55/defaults/main.yml#L25

Reply:
Yes, it doesn't have any effect at the moment.